src/globals/scss/_css--typography.scss

```diff
@@ -6,6 +6,39 @@
 @import 'css--reset';
 @import 'import-once';
 
+$type--css-classes: false;
+$type--reset: false;
+
+@import './vendor/@carbon/type/scss/classes';
+@import './vendor/@carbon/type/scss/reset';
+@import './vendor/@carbon/type/scss/scale';
+
+@mixin breakpoint($size, $given-breakpoints: $grid-breakpoints) {
```

Comment:
@joshblack Wondering if you have any ideas to cope with this kind of `@mixin` name conflicts. I'm feeling it's easier/safer if `cabron-elements` can rename those. Thanks!